### DISCUSSION TITLE
No module named cflib

### DISCUSSION THREAD

**Discussion by WolfscorpioTest** [bitcraze/discussions#1274] [Q&A] [answered]
When I'm trying to run the sbs_connect_log_param.py example it keeps saying no module named cflib. When I type pip3 install cflib, it says requirement already satisfied. How can I fix this issue?

**Comment by ToveRumar** [comment]
Hi! 
This looks like this might be an issue with the set up. 
Could you give me some more information regarding your environment. 
Are you running in a virtual environment? 
What python version are you using and what operative system are you running?

**Reply by WolfscorpioTest** [reply]
Hi Thank you for the response! I tried running it in both virtual environment and not. I am running python 3.11.8 and I am using ubuntu. I tried uninstalling and reinstalling everything as well.

**Reply by ToveRumar** [reply]
Hi!

Are you installing  cflib from source or as a lib via pip?

Have you made sure that the cflib package ends up in your python environment?
I.e that cflib can be found in site-packages in the environment lib directory, and that this is the correct python installation that is running the example.

**Reply by WolfscorpioTest** [reply, marked as answer]
Hi! It turns out it was some kind of problem with usb permissions and after I entered the commands in the usb permission install docs it worked.
